scripts/generate-prefecture-municipality-names.py

```
@@ -0,0 +1,193 @@
+    cur = con.cursor()
+    cur.execute("""
+        SELECT DISTINCT prefecture, city
+        FROM houjin
+        WHERE prefecture != '' AND city != ''
+        ORDER BY prefecture, city
+    """)
+    raw = cur.fetchall()
+    con.close()
+
+    rows = []
+    seen_names: set[str] = set()   # 重複チェック（同一名が複数府県に存在する場合は別エントリ）
```

Ruff (0.15.1)

[error] Local variable `seen_names` is assigned to but never used

Remove assignment to unused variable `seen_names`

(F841)